vsintegration/src/FSharp.Editor/Common/Extensions.fs

```diff
@@ -223,3 +262,24 @@ module Exception =
         | _ -> root
         |> flattenInner
         |> String.concat " ---> "
+
+[<RequireQualifiedAccess>]
+module Dict = 
+    open System.Collections.Generic
+
+    let add key value (dict: #IDictionary<_,_>) =
```

Comment:
I guess intention here was if this function was going to be used for ConcurrentDictionary too